packages/project/package.json

```diff
@@ -4,12 +4,14 @@
   "description": "Webstudio Project",
   "author": "Webstudio <github@webstudio.is>",
   "homepage": "https://webstudio.is",
+  "type": "module",
   "scripts": {
     "typecheck": "tsc --noEmit",
-    "test": "jest",
+    "test": "NODE_OPTIONS=--experimental-vm-modules jest",
     "checks": "yarn typecheck && yarn lint && yarn test",
     "dev": "tsup --watch",
```

Comment:
They did different things before too. tsup bundles code. tsc produce transpiled modules. But yeah, ideally have single tool. But probably we can worry about this later. As only tests will be affected.

Reply:
Eventually we would not need watch node if we switch to source code as entry point.

Reply:
are we making yarn dev or yarn test slower by introducing more artifacts?

Reply:
yep, tsc is slow and now it's slower but only for a few packages, not all.

Reply:
My main concern related to this right now is that the `dev` script doesn't actually build all of the required artifacts. So this script becomes sensitive to what other scripts you might run. Like, it might be enough when you just work on the app, but it might not be enough for whatever needs CJS artifacts (jest?). It's raising a mental overhear around this because whoever runs this script needs to know that this is the case - so the overall experience is not seamless.